### PR TITLE
feat: add `orientation` prop on the `RadioGroup` and `CheckboxGroup` components

### PR DESCRIPTION
**Pull request by luisFilipePT**
fix: #1824 

## Screenshots

image

image

image

image

## Changes proposed in this PR:
- Add optional orientation prop to radio group and checkbox group (default vertical)
- Possible values ['horizontal', 'vertical]

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/CheckboxGroup/index.d.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface CheckboxGroupProps extends BaseProps {
     error?: ReactNode;
     onChange?: (values: string[]) => void;
     id?: string;
+    orientation?: string;
```
Comment (LeandroTorresSicilia) [inline src/components/CheckboxGroup/index.d.ts:19]: here we need to be a little bit more specific:
```orientation?: 'vertical' | 'horizontal'```

File: src/components/CheckboxGroup/readme.md [hunk 2]
```diff
@@ -272,3 +272,46 @@ const CheckBoxGroupProfile = () => {
...
+    { value: 'checkboxTwo', label: 'Checkbox Two', disabled: false },
+    { value: 'checkboxThree', label: 'Checkbox Three', disabled: false },
+];
+
+class CheckboxGroupTry extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { values: [] };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
```
Comment (LeandroTorresSicilia) [inline src/components/CheckboxGroup/readme.md:293]: for new examples, we are using functions with hooks

File: src/components/RadioGroup/index.d.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export interface RadioGroupProps extends BaseProps {
     options?: RadioOption[];
     error?: ReactNode;
     id?: string;
+    orientation?: string;
```
Comment (LeandroTorresSicilia) [inline src/components/RadioGroup/index.d.ts:13]: same ^^

File: src/components/RadioGroup/readme.md [hunk 4]
```diff
@@ -168,3 +168,47 @@ class ErrorRadioGroup extends React.Component {
...
+];
+
+class SimpleRadioGroup extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: undefined,
+        };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
```
Comment (LeandroTorresSicilia) [inline src/components/RadioGroup/readme.md:191]: same ^^ let's use functional component instead

File: src/components/CheckboxGroup/readme.md (hunk 2 above)
Comment (luisFilipePT) [reply src/components/CheckboxGroup/readme.md]: ok, done with arrow function, not sure how you want to do it, not a lot of references. If you prefer "normal" functions just let me know.

File: src/components/RadioGroup/readme.md (hunk 4 above)
Comment (luisFilipePT) [reply src/components/RadioGroup/readme.md]: same ^^reply 😄

File: src/components/CheckboxGroup/index.d.ts (hunk 1 above)
Comment (luisFilipePT) [reply src/components/CheckboxGroup/index.d.ts]: Done https://github.com/nexxtway/react-rainbow/pull/1834/commits/58844df7f0f58f689898d98d6b10756f650c42bd#diff-2d2c8927e8faade40badec10e495da5d

File: src/components/RadioGroup/index.d.ts (hunk 3 above)
Comment (luisFilipePT) [reply src/components/RadioGroup/index.d.ts]: Done https://github.com/nexxtway/react-rainbow/commit/58844df7f0f58f689898d98d6b10756f650c42bd#diff-13dd6b49e6ca720fa18ef25b330ee6ab

**Review by TahimiLeonBravo — APPROVED** [review]

**Review by LeandroTorresSicilia — APPROVED** [review]
@luisFilipePT Good job and thanks for your contribution

**Pull request merged**